.github/actions/setup/action.yml

```diff
@@ -31,5 +31,13 @@ runs:
         restore-keys: |
           ${{ runner.os }}-puppeteer-          
 
+    - name: Disable AppArmor
+      # Ubuntu >= 23 has AppArmor enabled by default, which breaks Puppeteer.
```

Comment:
🤔 hmm -- glimmer-vm didn't need to do this
HMMMM

Reply:
Has the CI been updated from Ubuntu 22 to 24 yet? It is gradually rolling out

Reply:
ye: https://github.com/glimmerjs/glimmer-vm/actions/runs/12798438153/job/35682484023#step:1:4

Reply:
I believe this is why https://github.com/glimmerjs/glimmer-vm/blob/237c1e02a17d8823dcfb41e1fbd5f8c5108504ce/bin/run-tests.mjs#L47

This is the exact error:
> Running Chrome headless: http://localhost:13141/?
[2084:2084:0117/221855.488835:FATAL:zygote_host_impl_linux.cc(128)] No usable sandbox! If you are running on Ubuntu 23.10+ or another Linux distro that has disabled unprivileged user namespaces with AppArmor, see https://chromium.googlesource.com/chromium/src/+/main/docs/security/apparmor-userns-restrictions.md. Otherwise see https://chromium.googlesource.com/chromium/src/+/main/docs/linux/suid_sandbox_development.md for more information on developing with the (older) SUID sandbox. If you want to live dangerously and need an immediate workaround, you can try using --no-sandbox